### PR TITLE
Add default tactic and stategy handling, also add shortcut for them.

### PR DESCRIPTION
**Pull request by PhiBabin**
Related to issue #81
Add buttons for default tactic and default strategies. They also can be call with shortcut, such as Q et W. I added a shortcut for stop (S).

### CODE REVIEW

File: View/StrategyCtrView.py [hunk 1]
```diff
@@ -196,6 +221,27 @@ def hideEvent(self, event):
...
 
+    def keyPressEvent(self, event):
+        key = event.key()
+
+        page_id = self.page_controller.currentIndex()
+        if page_id == 1: # Strategy
+            if key == QtCore.Qt.Key_Q:
+                self.send_quick_strat1()
+            if key == QtCore.Qt.Key_W:
+                    self.send_quick_strat2()
```
Comment (PhiBabin) [inline View/StrategyCtrView.py]: oups

File: View/StrategyCtrView.py [hunk 2]
```diff
@@ -247,18 +326,37 @@ def refresh_strat(self, strats):
...
     def send_strat(self):
         strat = str(self.selectStrat.currentText())
         if not strat == 'Aucune Stratégie disponible':
             self.parent.model_dataout.send_strategy(strat, self.parent.get_team_color())
 
-    def send_tactic(self):
-        id_bot = int(self.selectRobot.currentText())
+    def send_apply_tactic(self):
         tactic = str(self.selectTactic.currentText())
+        if not tactic == 'Aucune Tactique disponible':
```
Comment (TwistedSim) [inline View/StrategyCtrView.py]: I'm not sure what is the operation priority here. Can we make this explicit? Also, a comparison with a hardcoded string like that will be hard to maintain.

File: View/StrategyCtrView.py [hunk 3]
```diff
@@ -247,18 +326,37 @@ def refresh_strat(self, strats):
...
+            self.send_tactic(tactic)
+
+    def send_tactic(self, tactic: str):
+        id_bot = int(self.selectRobot.currentText())
         args = str(self.argumentsLine.text()).split()
         target = self.parent.model_dataout.target
-        if not tactic == 'Aucune Tactique disponible':
-            self.parent.model_dataout.send_tactic(id_bot, self.parent.get_team_color(), tactic=tactic, target=target, args=args)
+        self.parent.model_dataout.send_tactic(id_bot, self.parent.get_team_color(), tactic=tactic, target=target, args=args)
```
Comment (TwistedSim) [inline View/StrategyCtrView.py]: args=args is not really clear. which informations are transmit?

**Review by TwistedSim — APPROVED** [review]

File: View/StrategyCtrView.py (hunk 2 above)
Comment (PhiBabin) [reply View/StrategyCtrView.py]: You know `!=` cost a lot of money

File: View/StrategyCtrView.py (hunk 3 above)
Comment (PhiBabin) [reply View/StrategyCtrView.py]: They are the optional argument field that's pass to the tactic. (The empty field over the "Appliquez'' button). For example, in GoToPosition the argument is the cruise speed.
It's one the stuff that I don't really like about the StrategyIA<=>UiDebug relation.

**Review by SimLemay — APPROVED** [review]

**Review by SimLemay — APPROVED** [review]

**Review by SimLemay — APPROVED** [review]

**Pull request merged**